src/client/api.cpp

```diff
@@ -61,9 +63,122 @@ void shutdownPRT() {
 	PRTContext::shutdown();
 }
 
+void getAnnotations(const prt::RuleFileInfo::Entry* attribute, std::vector<std::vector<py::object>>& annotations,
+                    bool& hidden) {
+	for (size_t f = 0; f < attribute->getNumAnnotations(); f++) {
+		const prt::Annotation* annot = attribute->getAnnotation(f);
+		if (std::wcscmp(annot->getName(), ANNOT_HIDDEN) == 0) {
+			hidden = true;
+			break;
+		}
+
+		if (std::wcsncmp(annot->getName(), L"@", 1) == 0) {
+			std::vector<py::object> annotationPyList;
+			annotationPyList.push_back(py::cast(annot->getName()));
+
+			for (size_t u = 0; u < annot->getNumArguments(); u++) {
+				py::object annotationValue;
+				const prt::AnnotationArgument* annotationArg = annot->getArgument(u);
+				const prt::AnnotationArgumentType annotationValueType = annotationArg->getType();
+
+				if (annotationValueType == prt::AAT_STR)
+					annotationValue = py::cast(annotationArg->getStr());
+				else if (annotationValueType == prt::AAT_BOOL)
+					annotationValue = py::cast(annotationArg->getBool());
+				else if (annotationValueType == prt::AAT_FLOAT)
+					annotationValue = py::cast(annotationArg->getFloat());
+				else
+					annotationValue = py::cast("UNKNOWN_PARAMETER_VALUE_TYPE");
+
+				py::list annotationParameters;
+				if (std::wcscmp(annotationArg->getKey(), NO_KEY) == 0)
+					annotationParameters.append(py::cast(NO_KEY_PY));

```

Comment:
Would it be possible to make this NO_KEY visible in the API as Python "constant" (or enum?), so that users do not have to care about the actual string value? I.e. users just can compare the key to pyprt.NO_KEY to see if they need to process it...